tikv-configuration-file.md

```diff
@@ -386,7 +386,7 @@ Configuration items related to Raftstore
 + Default value: `"3s"`
 + Minimum value: `0`
 
-### `raftstore.hibernate-regions` (**Experimental**)
+### `hibernate-regions` (**Experimental**)
```

Comment:
same comment as in https://github.com/pingcap/docs-cn/pull/5605/files#r582574763